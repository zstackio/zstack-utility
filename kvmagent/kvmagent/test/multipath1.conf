defaults {
  user_friendly_names yes
  find_multipaths yes
}

devices {
  device {
    vendor "DGC"
    product ".*"
    product_blacklist "LUNZ"
    path_grouping_policy "group_by_prio"
    path_checker "emc_clariion"
    features "1 queue_if_no_path"
    hardware_handler "1 emc"
    prio "emc"
    failback immediate
    rr_weight "uniform"
    no_path_retry 60
    retain_attached_hw_handler yes
    detect_prio yes
    detect_path_checker yes
  }
}
blacklist {
  wwid *
}