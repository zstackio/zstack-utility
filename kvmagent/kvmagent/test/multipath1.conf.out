defaults {
    find_multipaths yes
    user_friendly_names yes
}
devices {
device {
    detect_path_checker yes
    detect_prio yes
    failback immediate
    hardware_handler 1 emc
    no_path_retry 60
    path_checker emc_clariion
    path_grouping_policy group_by_prio
    prio emc
    product .*
    product_blacklist LUNZ
    retain_attached_hw_handler yes
    rr_weight uniform
    vendor DGC
}
device {
    features 0
    no_path_retry fail
    product *
    vendor *
}

}
blacklist {
    wwid *
}